client/navigation/style.scss

```diff
@@ -28,14 +42,10 @@
 		box-sizing: border-box;
 	}
 
-	&:not(.is-folded) {
-		#wpcontent,
-		#wpfooter {
-			margin-left: $navigation-width;
-
-			@media ( max-width: 960px ) {
-				margin-left: 0;
-			}
+	&.is-expanded {
```

Comment:
This seems like a bit of a code smell.  Do we need the additional class here?  It seems like a binary state selector should only contain a single selector that is toggled on/off unless we plan on introducing more states for the navigation in the future.

Reply:
The issue here is that we didn't have any way to apply styles specific to the expanded state that didn't also apply during the loading state (no body classes applied yet), which is what is required here I believe. 

Since this issue is specifically calling for styles that impact only the loading state, the only way to prevent them from influencing the display _after_ loading, and while the nav is expanded, is to have another class to key off of. 

Does that make sense?

Reply:
Having said that, I'm actually wondering now if there's another body class that already exists that I could utilize. I'll look into it real quick o.0

Reply:
There is a `woocommerce-admin-is-loading` class, but that will hide other elements on the page.

>Since this issue is specifically calling for styles that impact only the loading state

Makes sense.  Let me tinker with this for a few minutes, but at a glance it does seem like the way you've handled it here may be the most straightforward solution.

Reply:
I thought I could potentially use the `no-js` body class to key off of, but it seems there's way too much of a delay between the class being removed and the `<Nav />` component being mounted. 

An alternative solution may be to add a body class via `body_class()` server-side that we then take away when the component is mounted (similar to `no-js` but not removed until we're ready). This would simplify the logic overall, but perhaps be a little less intuitive.

Reply:
Just saw your earlier comment. It doesn't look like the `woocommerce-admin-is-loading` class is attached in a way that will be helpful. Let me know if you come across anything in your tinkerin'!

Reply:
The `no-js` solution is a nice workaround, but I agree that it's not so intuitive and will complicate things further.  I think the solution you've created here is the best for now. 👍 a